cfme/tests/test_rest.py

```diff
@@ -374,13 +374,446 @@ def test_vm_add_lifecycle_event(request, setup_a_provider, rest_api, vm, from_de
 }
 
 
+@pytest.fixture(scope="module")
+def users_data():
+    name = fauxfactory.gen_alphanumeric()
+    result = [{
+        "name": "name_{}".format(name),
+        "userid": "userid_{}".format(name),
+        "email": "{}@local.com".format(name),
+    } for _ in range(1, 5)]
```

Comment:
According to the docs (http://manageiq.org/documentation/development/rest_api/models/users/)
Password is optional, so I think it should work without it. 
About names, you are right. Fixed it.